### PR TITLE
Stubs for stripe

### PR DESCRIPTION
**Pull request by exemplary-citizen**
A library that provides convenient access to the Stripe API from applications. It includes a pre-defined set of classes for API resources that initialize themselves dynamically from API responses.

I started with stubgen, then cleaned up to address missing functions. Testing plan is to keep an eye on CI and fix any issues that arise.

cc: @JukkaL @srittau

### CODE REVIEW

File: stubs/stripe/stripe/api_resources/abstract/api_resource.pyi [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from typing import Any
+
+from stripe import api_requestor as api_requestor, error as error, util as util
```
Comment (JukkaL) [inline stubs/stripe/stripe/api_resources/abstract/api_resource.pyi:3]: I wonder if these `util as util` imports can be removed? I suspect that they are internal and thus don't need to be exposed. They are generated in many files.

**Review by JukkaL — APPROVED** [review]
Great, looks good! Just one optional suggestion about possibly redundant definitions.

File: stubs/stripe/stripe/api_resources/abstract/api_resource.pyi (hunk 1 above)
Comment (JelleZijlstra) [reply stubs/stripe/stripe/api_resources/abstract/api_resource.pyi]: Yes, this is a stubgen quirk; it is overzealous in generating re-exports. We ran into this with a few other packages recently too but I haven't looked into whether there's a concrete change we can make.

Comment (srittau) [reply stubs/stripe/stripe/api_resources/abstract/api_resource.pyi]: I was contemplating only generating re-exports in `__init__.pyi` or if there is an explicit re-export in the implementation, but so far I haven't found time to work on this.

**Pull request merged**